admin/src/components/Collections.js

```diff
@@ -200,12 +112,10 @@ const Collections = ({ updateCredentials }) => {
 
     if (error) errorNotifications(res)
     else {
-      // Start watching collections that have pending tasks
-      collections.map(col => {
-        if (col.isIndexing) {
-          watchTasks({ collection: col.collection })
-        }
-      })
+      const isIndexing = collections.find(col => col.isIndexing === true)
+
+      if (!isIndexing) setRealTimeReports(false)
+      else setRealTimeReports(true)
```

Comment:
Can't you just do `setRealTimeReports(isIndexing)` ?

Reply:
Absolutely 😂 I was tired